queue_job/models/queue_job.py

```diff
@@ -231,6 +232,12 @@ def _compute_graph_jobs_count(self):
 
     @api.model_create_multi
     def create(self, vals_list):
+        test_mode = getattr(threading.current_thread(), 'testing', False) or self.env.registry.in_test_mode()
```

Comment:
why both? in which case `getattr(threading.current_thread(), 'testing', False) != self.env.registry.in_test_mode()`?